### DISCUSSION TITLE
No output after commit 84d9015 on Android

### DISCUSSION THREAD

**Discussion by ShouNichi** [ggml-org/llama.cpp#234] [General]
When `git checkout 84d9015` and `make`, there will be no output (only the model loading message) in termux.
`git checkout 63fd76f` will produce a fully-functional binary.

**Comment by Green-Sky** [comment]
47857e564c218a2c38346d0cdd94314632878fcb should have fixed that again.

**Reply by gjmulder** [reply]
I moved this into issues. See #237.
